page-toc-filter.gemspec

```diff
@@ -17,9 +17,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'html-pipeline', '~> 2.0'
-  spec.add_dependency 'nokogiri', '~> 1.6.4'
+  spec.add_development_dependency 'nokogiri', '~> 1.8.0'
+  spec.add_development_dependency 'markup_validity'
```

Comment:
As @gwincr11 said in the comment below, it looks like this is in fact a [dev dep](https://github.com/gjtorikian/page-toc-filter/blob/master/test/test_helper.rb#L5).